frontstage/filters/subject_filter.py

```diff
@@ -0,0 +1,8 @@
+def subject_filter(subject):
+    """
+    Filter to format subject if subject is empty or has only spaces.
+    """
+    if subject is None or len(subject.strip()) < 1:
+        subject = '[no subject]'
```

Comment:
We're actually not testing this functionality, might be an idea to add that test in